servers/visual/shader_language.cpp

```diff
@@ -2498,7 +2498,7 @@ bool ShaderLanguage::_get_completable_identifier(BlockNode *p_block, CompletionT
 
 	identifier = StringName();
 
-	TkPos pos;
+	TkPos pos = {0, 0};
```

Comment:
`clang-format` requires this style here: https://travis-ci.org/godotengine/godot/jobs/445889428

Reply:
I'm a slow learner. Sorry for that :)